perf-runner/src/result-processor.mjs

```diff
@@ -0,0 +1,137 @@
+import { readFileSync } from 'fs';
```

Comment:
It would be awesome to see this as an incremental addition to the existing `compare` command. Especially awesome to see it output the node version and other things you have in here.